docs/build/program/run.md

```diff
@@ -77,62 +77,53 @@ If you subsequently lose internet connectivity, but stay connected to LAN or WAN
 
 ## Run code on-machine
 
-If you run [PID control loops](https://en.wikipedia.org/wiki/PID_controller) or your machines have intermittent or no network connectivity, you can ensure this does not interfere with the code's execution by running the control code on the same board that is running `viam-server`.
+You can run SDK code directly on your machine.
+If you run [PID control loops](https://en.wikipedia.org/wiki/PID_controller) or your machines have intermittent or no network connectivity, you can ensure lags in communication do not interfere with the machine's performance by running the control code on the same board that is running `viam-server`.
+Running everything on one machine is also convenient if you have a machine (for example, an air quality sensor) that runs all the time, and you don't want to have to connect to it from a separate computer constantly.
 
-{{<imgproc src="/build/program/on-robot.png" resize="800x" declaredimensions=true alt="A client running on a machine">}}
+The script you run on-machine is the same as the script you [run remotely](#run-code-remotely) or on a local network.
+When the connection code from the [code sample tab](/build/program/#hello-world-the-code-sample-tab) executes, it creates a [client session](/build/program/apis/sessions/) connected to your machine using the [most efficient route](/build/program/connectivity/).
+Because the code is running on the same machine as `viam-server`, the favored route for commands is automatically over localhost.
 
-When connecting to a machine using the connection code from the [code sample tab](/build/program/#hello-world-the-code-sample-tab), a [client session](/build/program/apis/sessions/) automatically uses the [most efficient route](/build/program/connectivity/) to connect to your machine, which means the favored route for commands will be over localhost.
-
-## Run code automatically
-
-If you want to run your code on-machine automatically when your machine boots, you can configure Viam to run your code as a [process](/build/configure/#processes).
-
-To be able to run your code from your board, you need to install the relevant SDK as well as other required dependencies:
+Install the appropriate programming language and [Viam SDK](/sdks/) on your machine and run a command to execute the program in the terminal of that machine instead of from a separate computer:
 
 {{< tabs >}}
 {{% tab name="Python" %}}
 
-1. [`ssh` into your board](/get-started/installation/prepare/rpi-setup/#connect-with-ssh) and install `pip`:
```

Comment:
We do need to move this to the process page. The whole installing into a folder thing is not intuitive. I'd consider putting that into an include to include it in both pages maybe even I think it's helpful

Reply:
It doesn't need to be installed into a given folder though. Virtual environments may affect this but I've never installed it into the same folder and it has worked!

Reply:
I found where we discussed this: https://viaminc.slack.com/archives/C02SYTDQ56X/p1683924608761639

Reply:
Ahhh I'm sorry it turns out the process I was running doesn't actually use Viam Python SDK methods--it's a non-Viam Python script. Will amend, and I will also add a note to make it clear that these steps are only necessary if you're running a **Viam SDK** script **as a process**, not just if you're running any Python script as a process.

Reply:
Done

Reply:
I added it to the processes page but I don't think it needs to be here too since this page no longer covers any instructions for setting up a process; folks need to follow the link anyway.